### ISSUE TITLE
Incorrect WCAG level for Abbreviation section

### ISSUE THREAD

**Issue by Nay-max** [daisy/kb#67]
Observed incorrect WCAG level AA mentioned for 3.1.4 Abbreviations section. It should be WCAG level AAA.

https://kb.daisy.org/publishing/docs/html/abbr.html

![image](https://github.com/daisy/kb/assets/149762133/0280ba90-ca03-4e60-9fde-c5f9451d6d49)

Please review and do the needful

**Comment by mattgarrish** [comment]
Yes, I've fixed it now in https://github.com/daisy/kb/commit/9448e2ade641bf990949b5d5c9cdd7b246b89f0c

Thanks for the report!
